metaflow/datastore/content_addressed_store.py

```diff
@@ -189,3 +191,10 @@ def _pack_v1(self, blob):
     def _unpack_v1(self, blob):
         with gzip.GzipFile(fileobj=blob, mode='rb') as f:
             return f.read()
+
+class BlobCache(object):
```

Comment:
I would prefer if load and store were symmetrical or named differently. It is confusing here to have a single load and a store (load/store are symmetrical but they are not in their use).

Reply:
sure, we can change the names

Reply:
Changed names to `load_key` and `store_keys`